_includes/manuals/1.10/4.4.8.3_prov_windows.md

```diff
@@ -0,0 +1,160 @@
+Provisioning Windows is a two step process. The first step, creating Installation Media, is not discussed here but can be found in the [Wiki](http://projects.theforeman.org/projects/foreman/wiki/Tutorials). The second step, setting up templates, is discussed in the following section.
+
+The necessary boot files are are later downloaded by automatically by smart-proxy.
+
+##### Tasks break down
+- Change / add a new Architecture and OS
+- Edit provision templates
+- Add installation media
+- Edit partition table
+- Add parameters
+- Link provisioning templates to OS
+
+##### Architecture and OS
+In _Hosts -> Architectures_ add a new architecture:
+
+- Name: `x64`
+
+Add a new OS in _Hosts -> Operating systems_ if needed.
+If you already have Windows hosts and with Puppet installed, the correct OS and architecture has been auto created already.
+This example covers Windows 8.1 / Windows Server 2012R2.
+
+![Add new OS](/static/images/screenshots/4.4.8.3_prov_win_os.png "Adding Windows 8 OS in Foreman")
+
+- Name: `windows`
+- Major: `6`
+- Minor: `3`
+- OS family: `windows`
+- Description: `Windows8`
+- Root password hash: `Base64`
+- Architectures: `x64`
+
+Take special care to set __Root password hash = `Base64`__. The templates do not render correctly if this is set otherwise.
+Changing the encoding does not [apply do existing hosts](/manuals/{{page.version}}/index.html#3.5.2ConfigurationOptions)
+
+##### Add provision templates
+Head to _Hosts -> Provisioning Templates_ and edit the templates starting with `WAIK` to your needs. Make sure to get the latest version of the WAIK templates from the [community templates project](https://github.com/theforeman/community-templates)
+Assign each of those templates to your Windows OS (does not apply to snippets).
+The naming of the templates is a suggestion and up to you. This does __not__ apply to snippets! There, the name is important.
+
+__Note__ You can find more info about [Foreman Template Writing in the Wiki](http://projects.theforeman.org/projects/foreman/wiki/TemplateWriting)
+
+##### Required templates
+###### WAIK Finish
```

Comment:
These headings are actually smaller than the text below because they're at six levels.  Perhaps use numbered lists with `1. WAIK Finish`, then indent the name/kind by four spaces.